### PR TITLE
Create config.yml

### PR DESCRIPTION
**Pull request by VaaibhaviSingh**
Added a new file config.yml as mentioned in issue #33

### CODE REVIEW

File: .github/config.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: |
+  Thanks for opening your first issue here! Please follow the issue template to help us help you 👍🎉😄
+  If you have screenshots to share demonstrating the issue, that's really helpful! 📸
+
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+
+# Comment to be posted to on PRs from first time contributors in your repository
+newPRWelcomeComment: |
+ Thanks for opening this pull request! `Dangerbot` will test out your code and reply in a bit with some pointers and requests.
```
Comment (jywarren) [inline .github/config.yml]: Actually for this repository I don't think we use Dangerbot! So maybe that bit could be removed...

File: .github/config.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ There may be some errors, **but don't worry!** We're here to help! 👍🎉😄
+
+# Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
+
+# Comment to be posted to on pull requests merged by a first time user
+firstPRMergeComment: |
+ Congrats on merging your first pull request! 🙌🎉⚡️ 
+ Your code will likely be published to PublicLab.org in the next few days.
```
Comment (jywarren) [inline .github/config.yml]: And here, I think maybe we can remove the part about PublicLab.org -- or replace it with `code.publiclab.org` ? Thank you!!!

Comment (VaaibhaviSingh) [reply .github/config.yml]: I'll work to correct this! 😃

**Comment by VaaibhaviSingh** [comment]
Submitted a new PR(#45) with the mentioned changes!
Thank you : )

**Comment by gauravano** [comment]
@VaaibhaviSingh please close this issue. Thanks!

**Pull request closed without merging**